nts-pool-ke/src/lib.rs

```diff
@@ -192,6 +209,208 @@ async fn pool_key_exchange_server(
     }
 }
 
+async fn handle_client(
+    client_stream: tokio::net::TcpStream,
+    config: Arc<rustls::ServerConfig>,
+    certificate_authority: Arc<[rustls::Certificate]>,
+) -> Result<(), KeyExchangeError> {
```

Comment:
in general here too there is the question of how to report errors back to the NTS client. E.g. if the pool <-> server has an error, that must ultimately be reported to the client. Should that forward the error from the server, or just always return an InternalServerError?

Reply:
I think this can be made into a seperate issue; right now this is "proof of concept" code.

Reply:
filed https://github.com/pendulum-project/ntpd-rs/issues/1246